packages/block-library/src/image/edit.native.js

```diff
@@ -153,6 +158,14 @@ class ImageEdit extends React.Component {
 		setAttributes( { url: mediaUrl, id: mediaId } );
 	}
 
+	iconWithUpdatedFillColor( color, icon ) {
```

Comment:
I see that we use this function from 3 places already and every file uses its own copy, I was thinking what would be a better place to put this? maybe @Tug can assist us on this.

Reply:
Well yeah, React is all about reusable components. Don't hesitate to add mobile specific ones in `@wordpress/components`, if it's a mobile specific one you can put it in a specific directory.

In this particular case you can probably reuse the `Icon` component given a few tweaks to the `Icon` component:
```jsx
<Icon icon={ svgIcon } style={ { fill: color } } />
```

Reply:
You could have your colors in css classes and won't even need to tweak the `Icon` component:
```
<Icon icon={ svgIcon } className="icon-fill-color" />
```

Reply:
Hey @pinarol, @Tug I managed to reuse this logic by creating `Icon` component for RN.
I didn't reuse it from the GB web as it has a lot of code and imports that we don't want to use.

Thanks, @Tug, for guiding to the right direction :)